coalib/results/TextPosition.py

```diff
@@ -3,7 +3,6 @@
 
 
 @generate_repr('line', 'column')
-@generate_ordering('line', 'column')
```

Comment:
ah you implement it. Though the `__eq__` overload has to come before this commit :+1: